runtime/src/main/java/org/corfudb/runtime/collections/Table.java

```diff
@@ -72,6 +74,7 @@ public Table(@Nonnull final String namespace,
         this.corfuRuntime = corfuRuntime;
         this.namespace = namespace;
         this.fullyQualifiedTableName = fullyQualifiedTableName;
+        this.streamUUID = UUID.nameUUIDFromBytes(this.fullyQualifiedTableName.getBytes());
```

Comment:
can we use CorfuRuntime.getStreamID(tableName)?

Reply:
done.